### ISSUE TITLE
Teams > TeamsFederationConfiguration - config drifts

### ISSUE THREAD

**Issue by archeedev** [microsoft/Microsoft365DSC#2601]
It seems that the latest release - 1.22.1123.1 - fixed indeed the TeamsFederationConfiguration issue: #2576 

However, please correct me here if I am misinterpreting this, 
but I think we have an issue with evaluating drifts.

I see the external domains applied ok in the destination tenant - same amount of domains on both tenants, 
so there should probably be no drift warnings(?)

But I keep getting config. drift warnings - weird screenshot after it is redacted but they are the same domains (which is ok).

![image](https://user-images.githubusercontent.com/25514862/204293482-99a8969d-05b3-4688-9465-72f8b41fc18a.png)

Not yet sure exactly what is causing this, my working theory is that we might be comparing things wrongly, eg. (to follow the outcome from event viewer):

```
 <ParametersNotInDesiredState>
            <Param Name="AllowedDomains"><CurrentValue>gmail.com facebook.com ...
```

with

```
 <DesiredValues>
        <Param Name ="Identity">Global</Param>
        <Param Name ="AllowFederatedUsers">True</Param>
        <Param Name ="AllowedDomains">Domain=gmail.com,Domain=facebook.com,...
```

In other words: 
gmail.com <> Domain=gmail.com

**Comment by andikrueger** [comment]
Could you share an export of this setting? Maybe there is an issue on how the values are returned and compared.

**Comment by NikCharlebois** [comment]
Based on your screenshot it looks like your configuration defines the domains as:
```
AllowedDomains = @("Domain=northwind.com", "Domain=contoso.com")
```
With the **Domain=** prefix before. I would recommend removing those from the configuration and simply keeping the actual domain name.
